R/utils.R

```diff
@@ -173,8 +173,8 @@ anyUnnamed <- function(x) {
 }
 
 
-# Given a vector/list, returns a named vector (the labels will be blank).
-asNamedVector <- function(x) {
```

Comment:
A historical note: the reason this function was called `asNamedVector` is that a list is a kind of vector. For example:

```R
vector(mode = "list")
```

That said, I agree that the name `asNamed` is a little clearer, because it avoids the ambiguity of what "vector" means in R -- sometimes it means atomic vector, sometimes it has the broader meaning which includes lists.